### ISSUE TITLE
Price slider: can't move handles with the keyboard

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#1679]
![imatge](https://user-images.githubusercontent.com/3616980/73437563-dba7bc80-434c-11ea-8a11-08b05d3308f7.png)

**To Reproduce**

1. Create a post with the _All Products_ block and the _Price Filter_ with the option _Price Range_ > _Text_.
2. In the frontend, navigate to the price slider with the keyboard and try to move the handles with the arrow keys.
3. Notice it's not possible.

**Expected behavior**
The user should be able to move the handles when they are focused.
